assembl/static2/js/app/components/debate/survey/question.jsx

```diff
@@ -9,6 +9,7 @@ import { getIfPhaseCompletedByIdentifier } from '../../../utils/timeline';
 import { inviteUserToLogin, displayAlert } from '../../../utils/utilityManager';
 import createPostMutation from '../../../graphql/mutations/createPost.graphql';
 import { SMALL_SCREEN_WIDTH } from '../../../constants';
+import { getContentLocale } from '../../../reducers/rootReducer';
```

Comment:
Ahh, this will have a merge conflict with the branch I will be pushing. I will adapt on my end.